src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccessTokenServiceImpl.java

```diff
@@ -49,45 +48,39 @@
     @Transactional
     @Override
     public void requestNewAccessToken() {
-        AccessTokenDTO newToken = buildRequestAccessTokenOutput();
+        AccessTokenDTO newToken = callConcurEndpoint(
+                this::callRequestAccessTokenEndpoint, AccessTokenDTO.class);
```

Comment:
The naming here is confusing with the two functions both starting with "call".  The use of lambda expressions here with the names of the functions makes it appear that there are two calls being made when in fact there is only one call to the endpoint being performed to refresh the token.  I foresee more functions needing to be created that would be passed into "callConcurEndpoint" which would just increase the confusion. Is there a better way or a better naming convention to introduce here?

Reply:
Thanks for the feedback. I think trying to assign the method reference to a variable would just add more clutter in this case, but I do agree that the naming is causing confusion here. I'll work on cleaning up the method naming and, if needed, rework the method references so that they just focus on building the request.